src/org/usfirst/frc4904/robot/humaninterface/drivers/NathanGain.java

```diff
@@ -53,6 +54,9 @@ public void bindCommands() {
 		RobotMap.Component.driverXbox.dPad.right.whenPressed(new ChassisTurnAbsolute(RobotMap.Component.chassis, 90,
 			RobotMap.Component.navx, RobotMap.Component.chassisTurnMC));
 		RobotMap.Component.driverXbox.dPad.right.whenReleased(normalDrive);
+		Command autoShifter = new AutoShifter(RobotMap.Component.shifter, RobotMap.Component.leftWheelEncoder,
```

Comment:
Either place into a local field, or call new and .start on the same line without this temp variable